src/main/java/at/ac/tuwien/kr/alpha/config/CommandLineParser.java

```diff
@@ -127,6 +128,13 @@
 			.desc("activates the accumulator grounding strategy by disabling removal of instances from grounder memory in certain cases (default: "
 						+ SystemConfig.DEFAULT_GROUNDER_ACCUMULATOR_ENABLED + ")")
 			.build();
+	private static final Option OPT_ENABLE_RESTARTS = Option.builder("rs").longOpt("enableRestarts")
+		.desc("enable the usage of (dynamic and static) restarts (default: "
+			+ SystemConfig.DEFAULT_ENABLE_RESTARTS + ")")
+		.build();
+	private static final Option OPT_INITIAL_PHASE = Option.builder("ph").longOpt("initialPhase").hasArg(true).argName("initializer")
```

Comment:
This is a good point and touches on a larger issue: what we would need is to check whether the whole set of given options actually makes sense. For the time being, I'll change the description so that it becomes clear that this only works for the `VSIDS_WITH_PHASE_SAVING` heuristic.

Reply:
Added to issue #302 as a condition to check eventually.